e2e/tests/e2e_happy_path/HappyPath.spec.ts

```diff
@@ -289,7 +292,11 @@ async function runTask(task: string) {
     await quickOpenContainer.clickOnContainerItem(task);
     await quickOpenContainer.clickOnContainerItem('Continue without scanning the task output');
 }
-
+async function checkCodeNavigationWithContextMenu() {
+    await contextMenu.invokeContextMenuOnActiveElementWithKeys();
+    await contextMenu.waitContextMenuAndClickOnItem('Go to Definition');
+    console.log('please check the satus of the known issue: https://github.com/eclipse/che/issues/14520. If it is fixed, we have to return the test to previous state.');
```

Comment:
I'm not sure, if echoing this line every time the test is run is a good approach. We usually write that info as a comment to the method. But if you think this is better, I'm OK with that.

Reply:
I would put that message above the line ` // await editor.performKeyCombination(javaFileName, Key.chord(Key.CONTROL, Key.F12));` instead of spamming the log.

Reply:
This have been move out to readonly constants